app/controllers/articles_controller.rb

```diff
@@ -11,11 +11,12 @@ def feed
     skip_authorization
 
     @articles = Article.feed.order(published_at: :desc).page(params[:page].to_i).per(12)
+    @latest = request.path == latest_feed_path
```

Comment:
Does this need to be an instance variable?  My read is that it's local to this method.

Reply:
I use this as an easier check inside `feed.rss.builder` so I don't have to be calling `request.path` within the builder.  Is that an issue?